src/khoj/routers/helpers.py

```diff
@@ -607,9 +611,10 @@ async def send_message_to_model_wrapper(
         else conversation_config.max_prompt_size
     )
     tokenizer = conversation_config.tokenizer
+    model_type = conversation_config.model_type
     vision_available = conversation_config.vision_enabled
 
-    if conversation_config.model_type == "offline":
+    if model_type == "offline":
```

Comment:
We can check against the ChatModelOptions.ModelType enum here? There are other places in this file where `model_type == "offline"` string match is still being done too

```suggestion
    if model_type == ChatModelOptions.ModelType.OFFLINE:
```